packages/atlas-anatomogram-experiment-table/src/anatomogramExperimentTable.js

```diff
@@ -0,0 +1,313 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import Anatomogram from '@ebi-gene-expression-group/organ-anatomogram'
+import { TableManager } from '@ebi-gene-expression-group/atlas-experiment-table'
+import URI from 'urijs'
+
+const capitalizeFirstLetter = (str) => str.charAt(0).toUpperCase() + str.slice(1)
+
+const ontologyAccessionToOntologyUri = (accession) => {
+  return accession.startsWith(`EFO`) ?
+    `http://www.ebi.ac.uk/efo/${accession}` :
+    `http://purl.obolibrary.org/obo/${accession}`
+}
+
+
+class AnatomogramExperimentTable extends React.Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      selectedSpecies: props.species,
+      showIds: props.showIds,
+      experiments: props.experiments,
+      highlightIds: [],
+      selectIds: [],
+      selectAllIds: [],
+      selectedOrganId: ``

```

Comment:
Since we’re also changing the `resource` later via `setState`, let’s initialise it here for completeness:
```javascript
this.state = {
  ...,
  resource: props.resource
}
```